cmd/dummy/main.go

```diff
@@ -17,19 +18,24 @@ func main() {
 	persistencePtr := flag.String("persist", "", "directory to use for a database")
 	flag.Parse()
 
+	logger := log.NewTmLogger(os.Stdout)
+
 	// Create the application - in memory or persisted to disk
 	var app types.Application
 	if *persistencePtr == "" {
 		app = dummy.NewDummyApplication()
 	} else {
 		app = dummy.NewPersistentDummyApplication(*persistencePtr)
+		app.(*dummy.PersistentDummyApplication).SetLogger(log.With(logger, "module", "dummy"))
```

Comment:
it looks a bit ugly, but don't think that `SetLogger` should be a part of the `Application` interface.